app/assets/javascripts/admin/addon/components/site-settings/color.gjs

```diff
@@ -25,6 +27,16 @@ function RGBToHex(rgb) {
 }
 
 export default class Color extends Component {
+  get fallbackColor() {
+    if (SETTINGS_WITH_FALLBACK_COLORS.includes(this.args.setting?.setting)) {
+      const root = document.querySelector(":root");
+      const computedStyle = getComputedStyle(root);
+      const primaryColor = computedStyle.getPropertyValue("--primary");
+      return primaryColor;
```

Comment:
```suggestion
      return getComputedStyle(document.documentElement).getPropertyValue("--primary");
```

minor, i think it just makes it a bit briefer. seems like we use `documentElement` in other places in the code too

Reply:
great suggestion! implemented!